src/appMain/smartDeviceLink.ini

```diff
@@ -87,7 +87,7 @@ ReadDIDRequest = 5, 1
 ; Limitation for a number of GetVehicleData requests (the 1st value) per (the 2nd value) seconds
 GetVehicleDataRequest = 5, 1
 ; Limitation for a number of GetInteriorVehicleDataRequest requests (the 1st value) per (the 2nd value) seconds
-GetInteriorVehicleDataRequest = 5, 1
+GetInteriorVehicleDataRequest = 20, 1
```

Comment:
@LuxoftAKutsan in we have limitation per module type - we don't need this changes

Reply:
@yang1070 will rever it.